### PR TITLE
qa/workunits/cephadm/test_cephadm: some cleanup

### PR DESCRIPTION
**Pull request by liewegas**
- don't pull old images
- remove cruft for testing python2

### CODE REVIEW

**Review by sebastian-philipp — APPROVED** [review]

File: qa/workunits/cephadm/test_cephadm.sh [hunk 1]
```diff
@@ -160,7 +129,7 @@ function nfs_stop()
     # stop the running nfs server
     local units="nfs-server nfs-kernel-server"
     for unit in $units; do
-        if systemctl status $unit; then
+        if systemctl status $unit < /dev/null; then
```
Comment (mgfritch) [inline qa/workunits/cephadm/test_cephadm.sh:132]:
```suggestion
        if systemctl status $unit > /dev/null; then
```

**Review by mgfritch — APPROVED** [review]
one minor suggestion, otherwise lgtm!

File: qa/workunits/cephadm/test_cephadm.sh (hunk 1 above)
Comment (liewegas) [reply qa/workunits/cephadm/test_cephadm.sh:132]: I tried that first, but it doesn't prevent systemctl status from launching the pager; ``>`` does

Comment (mgfritch) [reply qa/workunits/cephadm/test_cephadm.sh:132]: maybe `systemctl --no-pager` ?

**Pull request merged**